hawc/apps/assessment/api.py

```diff
@@ -445,3 +445,12 @@ def get(self, request, casrn: str, format=None):
             tasks.get_dsstox_details.delay(casrn)
 
         return Response(data)
+
+
+class JobViewset(viewsets.ModelViewSet):
```

Comment:
if we have an assessment key on each job, can we add the standard assessment level permissions?

Reply:
Can we expect each job to have an associated assessment, or can assessment be null?

Reply:
ahh good point. sounds good. a uuid is pretty hard to guess, so we shouldn't be worried about permissions, and there's no really sensitive info here anyways